modules/combined/tests/elastic_thermal_patch/elastic_thermal_jacobian_rz_smp_test.i

```diff
@@ -75,76 +71,66 @@
     order = CONSTANT
     family = MONOMIAL
   [../]
-
-[] # AuxVariables
-
-[SolidMechanics]
-  [./solid]
-    disp_r = disp_x
-    disp_z = disp_y
-    temp = temp
-  [../]
 []
 
 [Kernels]
-
-#  [./body]
-#    type = BodyForce
-#    variable = disp_y
-#    value = 1
-#    function = body
-#  [../]
+  [./TensorMechanics]
```

Comment:
It would be best as we are converting these tests over to use the Modules/TensorMechanics/Master action, which can automatically set up all of the tensor components for output, which significantly cuts down on the size of these input files.